lib/checker.js

```diff
@@ -1,6 +1,7 @@
 var vowFs = require('vow-fs');
 var Vow = require('vow');
 var StringChecker = require('./string-checker');
+var extractJs = require('./extract-js');
```

Comment:
`HtmlFile` is just a helper for `extract` option, because we can't use `JsFile`, but errors must has a reference to some `file`.